packages/proto-codecs/scripts/package-types.ts

```diff
@@ -4,21 +4,20 @@
  */
 export type AvailablePackages =
```

Comment:
If this file is generated, it seems weird to have it in a scripts folder?

IMO

Reply:
I think it makes sense to have it inside `scripts` since it's intended to be used there. However, I understand the concern about its placement alongside non-generated files. I think we can place it inside a folder called `scripts/codegen`. wdyt?